src/components/Header/Header.tsx

```diff
@@ -22,39 +12,24 @@ export type HeaderProps = {
 };
 
 export function Header(props: HeaderProps) {
-	const { header, handleOidcAuth, isAuthenticated = false } = props;
-
-	const [brandTop, setBrandTop] = useState(DEFAULT_HEADER.brandTop);
-	const [homeLinkProps, setHomeLinkProps] = useState(
-		DEFAULT_HEADER.homeLinkProps
-	);
-	const [serviceTitle, setServiceTitle] = useState(DEFAULT_HEADER.serviceTitle);
-	const [operatorLogo, setOperatorLogo] = useState(DEFAULT_HEADER.operatorLogo);
-	const [quickAccessItems, setQuickAccessItems] = useState<Array<any>>([]);
-
-	useEffect(() => {
-		if (!header) {
-			return;
-		}
-		setBrandTop(header.brandTop || DEFAULT_HEADER.brandTop);
-		setHomeLinkProps(header.homeLinkProps || DEFAULT_HEADER.homeLinkProps);
-		setServiceTitle(header.serviceTitle || DEFAULT_HEADER.serviceTitle);
-		setOperatorLogo(header.operatorLogo || DEFAULT_HEADER.operatorLogo);
-	}, [header]);
-
-	useEffect(() => {
-		const others = header?.quickAccessItems || [];
-		setQuickAccessItems([
-			...others,
+	const { header, handleOidcAuth, isAuthenticated } = props;
+	const brandTop = header?.brandTop ?? DEFAULT_HEADER.brandTop;
+	const homeLinkProps = header?.homeLinkProps ?? DEFAULT_HEADER.homeLinkProps;
+	const serviceTitle = header?.serviceTitle ?? DEFAULT_HEADER.serviceTitle;
+	const operatorLogo = header?.operatorLogo ?? DEFAULT_HEADER.operatorLogo;
```

Comment:
Maybe it will clearer to extract this into function that will take header and return brandTop, homeLinkProps ...